packages/wonder-blocks-modal/components/modal-backdrop.js

```diff
@@ -26,14 +32,22 @@ export default class ModalBackdrop extends React.Component<Props> {
         // Was the lowest-level click target (`e.target`) the positioner element
         // (`e.currentTarget`)?
         if (e.target === e.currentTarget) {
-            this.props.onClickBackdrop();
+            this.props.onCloseModal();
         }
     };
 
     render() {
+        const children = this.props.children;
+        const clonedChildren = React.cloneElement(children, {
+            onClickCloseButton: () => {
+                children.props.onClickCloseButton();
```

Comment:
Do you have to check and see if this exists already, or not? Or maybe since it's required in the Flow props we're ok?

Reply:
Hmm good point! This worked out in testing because modal elements have a `onClickCloseButton` defined in `defaultProps`, but this won't work for wrapper components. I've added a test for it!